### PR TITLE
Fix: changed lottie lib

### PR DESCRIPTION
**Pull request by egor0798**
Lottie lib led to app stuck after displaying animation multiple time

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,24 +10,18 @@
   "scripts": {
     "start": "pnpm clean:build && concurrently \"pnpm start:renderer\" \"pnpm start:main\" && watch",
     "start:dev": "pnpm clean:build && concurrently \"pnpm start:renderer:dev\" \"pnpm start:main:dev\" && watch",
-
```
Comment (Asmadek) [inline package.json:13]: can you revert this changes?

**Review by Asmadek — APPROVED** [review]

**Review by pgolovkin — APPROVED** [review]

**Review by tuul-wq — APPROVED** [review]
